cli/cmd/compliance_aws.go

```diff
@@ -562,13 +568,20 @@ func init() {
 	// AWS report types: AWS_NIST_CSF, AWS_NIST_800-53_rev5, AWS_HIPAA, NIST_800-53_Rev4, LW_AWS_SEC_ADD_1_0,
 	//AWS_SOC_Rev2, AWS_PCI_DSS_3.2.1, AWS_CIS_S3, ISO_2700, SOC, AWS_CSA_CCM_4_0_5, PCI, AWS_Cyber_Essentials_2_2,
 	//AWS_ISO_27001:2013, AWS_CIS_14, AWS_CMMC_1.02, HIPAA, AWS_SOC_2, AWS_CIS_1_4_ISO_IEC_27002_2022, NIST_800-171_Rev2,
-	//AWS_NIST_800-171_rev2'
+	//AWS_NIST_800-171_rev2
 	complianceAwsGetReportCmd.Flags().StringVar(&compAwsCmdState.Type, "type", "AWS_CIS_14",
 		fmt.Sprintf(`report type to display, run 'lacework report-definitions list' for more information.
 valid types:%s`, prettyPrintReportTypes(api.AwsReportTypes())))
 
+	// mark report type flag as deprecated
+	err := complianceAwsGetReportCmd.Flags().MarkDeprecated("type", "use --report_name flag instead")
+	if err != nil {
+		log.Fatal("unable to deprecate flag '--type'")
```

Comment:
We do not use `log`, we have our own package `lwlogger`. use `cli.Log` instead.

Also, what are we trying to do with this log? 🤔 if you are just trying to check the error because lint tests are failing, then use `errcheckWARN()` instead.